common/scaffolding/typography.html

```diff
@@ -0,0 +1,39 @@
+---
+{
+	"title": "Typography",
+	"language": "en",
+	"altLangPage": "typographie.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-08-24"
+}
+---
+
+<h1>Headings</h1>
+<h1>This is an h1 heading element</h1>
```

Comment:
* [ ] Except for a few exception, a page should only contain one heading level 1
* [ ] Add the notice as described in my previous comment for the French page
* [ ] Apply the WAI Aria role paragraph

Reply:
Fixed and addressed in commit "Updated headings and added paragraph".